pydis_site/apps/api/migrations/0096_mailinglistseenitem_unique_list_and_hash.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 5.0 on 2023-12-13 08:00
+
+from django.db import migrations, models
+
+
```

Comment:
Wouldn't it be better to squash the schema migrations into one ?

Reply:
I've squashed the final two, however for the previous migrations I've intentionally opted to keep them separate because one is an elidable data migration. I suggest once this and the alternate account PR is merged we make a "big squash"

Reply:
Hello. My lawyer has advised me to squash the migrations as you requested. This is now live.

Reply:
Thanks !
Great lawyer btw, you should give him a raise.